core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -379,9 +476,9 @@ object ConsumerGroupCommand {
   }
 
   class ConsumerGroupCommandOptions(args: Array[String]) {
-    val ZkConnectDoc = "REQUIRED (only when using old consumer): The connection string for the zookeeper connection in the form host:port. " +
+    val ZkConnectDoc = "REQUIRED (unless new-consumer is used): The connection string for the zookeeper connection in the form host:port. " +
```

Comment:
Sure, I'll revert these changes. I'm OK with leaving the "required" text in since other tools have it too, but we need to better qualify them, as you mentioned. I may work on it separately later.